### ISSUE TITLE
[Report]: new stored injection category not building playbook for post endpoint

### ISSUE THREAD

**Issue by SuddenDevelopment** [apisec-inc/Support#204]
### Severity

Blocker (No Workaround Available)

### What happened?

I created a "Reflected POST Injection" private stored injection category in development.

I enabled it in the project

I added a POST endpoint in my API to test with. Verified it shows up in the imported schema.

When I tell it to create the playbook for the category it never shows up

**Comment by Mohammed-Luqman-Shareef** [comment]
@SuddenDevelopment, in which tenant in dev have you created this category?  I guess it is private category, so not visible in other tenants.  Let me know the tenant's name, I'll login into that and check.

**Comment by SuddenDevelopment** [comment]
in the development environment "unit test" project, I defined a response body on an endpoint and it still didnt pick it up when trying to create playbooks. Waiting on a fix to test

**Comment by Mohammed-Luqman-Shareef** [comment]
@SuddenDevelopment,  I guess you registered this API (https://security-categories.apisec.ai/v3/api-docs). It doesn't conain any POST method with a request body defined in it.  
I tried with our test API Netbanking (http://netbanking.apisec.ai:8080/v2/api-docs) and it playbooks for POST method got generated in this category.

Either you can correct the spec, or use the netbanking spec for your testing.

**Comment by SuddenDevelopment** [comment]
@Mohammed-Luqman-Shareef  it does it's here:
`    "/post_reflection_fail" : {
      "post" : {
        "deprecated" : false,
        "summary" : "POST reflection fail",
        "security" : [ ],
        "responses" : {
          "200" : {
            "description" : "success response",
            "content" : {
              "application/json" : {
                "schema" : {
                  "$ref" : "#/components/schemas/response_post_reflection"
                }
              }
            }
          }
        },
        "parameters" : [ ],
        "tags" : [ "xss" ]
      }
    },`

**Comment by SuddenDevelopment** [comment]
I figured this out thanks.  I needed a request body defined as a model. I got the playbook to create, fixed things in my test and it's ready to deploy, as a separate ticket. Moving on, thanks.
